example_app_generator/generate_action_mailer_specs.rb

```diff
@@ -14,15 +14,9 @@
       end
     end
   CODE
-  gsub_file 'config/initializers/action_mailer.rb',
-            /ExampleApp/,
-            Rails.application.class.parent.to_s
 
   copy_file 'spec/support/default_preview_path'
   chmod 'spec/support/default_preview_path', 0755
-  gsub_file 'spec/support/default_preview_path',
-            /ExampleApp/,
-            Rails.application.class.parent.to_s
```

Comment:
Why are we doing this? It should be replaced not removed no?

Reply:
We had errors like `undefined method 'parent' for ExampleApp::Application:Class `. I noticed that this gsub is not needed anymore.